detox/package.json

```diff
@@ -47,6 +48,7 @@
     "funpermaproxy": "^1.0.1",
     "get-port": "^2.1.0",
     "ini": "^1.3.4",
+    "jest-cli": "^26.2.2",
```

Comment:
Why is jest-cli a dependency? It might cause issues for users with different versions (wonky node_modules mess, where this supercedes the actual jest dependencies the users configured for themselves)

Reply:
Moved to peerDependencies.